packages/oca/src/legacy/resolver/oca.ts

```diff
@@ -203,93 +144,119 @@ export class OCABundle implements OCABundleType {
     return this.getOverlay<MetaOverlay>(OverlayType.Meta10, this.options.language)
   }
 
-  public get cardLayoutOverlay(): CardLayoutOverlay10 | CardLayoutOverlay11 | undefined {
-    return this.getOverlay(this.options?.cardOverlayType || CardOverlayType.CardLayout11)
+  public get brandingOverlay(): BrandingOverlay | LegacyBrandingOverlay | undefined {
+    return this.getOverlay(this.options?.brandingOverlayType || BrandingOverlayType.Branding10)
   }
 
   public buildOverlay(name: string, language: string): MetaOverlay {
```

Comment:
Not sure where this is used but seems like it's only building a `MetaOverlay` so the method name is confusing. I know it was added before this, but just making a comment. Nothing for you to change at this point